src/configure-balena-host-envvars.sh

```diff
@@ -0,0 +1,131 @@
+#!/bin/bash
+
+# If an envvar for a required host is not set, it is appended to the
+# Docker envvar file. This does have the caveat that potentially all
+# services end up with all hostname envvars, but they shouldn't be
+# conflicting so are simply redundant.
+# This *does* expect the `BALENA_TLD` envvar to be set to allow
+# the setting of hosts
+
+# List of host envvars used in services.
+# TBD:
+# * There are a lot of envvars here with different names pointing to
+#   the same host, we should standardise
```

Comment:
Yeah, please standardise these if they're going to be automatically set globally

Reply:
Yeah, this is one of the reasons this is a 'test' PR (along with all the other components atm). I'm not considering even going for review until they are, but there's a push for a demo next week, so PR-based images will do for now.